packages/client/hmi-client/src/page/Home.vue

```diff
@@ -54,52 +82,64 @@
 						</ul>
 					</div>
 				</div>
+				<div v-if="isLoadingProjects">
+					<p>
+						<Skeleton width="6rem" />
+					</p>
+					<div class="carousel">
+						<ul>
+							<li v-for="(i, index) in [0, 1, 2, 3, 4, 5]" class="card" :key="index">
+								<DocumentCard />
+							</li>
+						</ul>
+					</div>
+				</div>
 			</section>
-		</section>
-		<!-- modal window for showing selected document -->
-		<div
-			v-if="selectedDocument !== undefined"
-			class="selected-document-modal-mask"
-			@click="close()"
-		>
-			<div class="selected-document-modal" @click.stop>
-				<div class="modal-header">
-					<h4>{{ selectedDocument.title }}</h4>
-					<IconClose32 class="close-button" @click="close()" />
+			<!-- modal window for showing selected document -->
+			<div
+				v-if="selectedDocument !== undefined"
+				class="selected-document-modal-mask"
+				@click="close()"
+			>
+				<div class="selected-document-modal" @click.stop>
+					<div class="modal-header">
+						<h4>{{ selectedDocument.title }}</h4>
+						<IconClose32 class="close-button" @click="close()" />
+					</div>
+					<selected-document-pane
+						class="selected-document-pane"
+						:selected-document="selectedDocument"
+						@close="close()"
+					/>
 				</div>
-				<selected-document-pane
-					class="selected-document-pane"
-					:selected-document="selectedDocument"
-					@close="close()"
-				/>
 			</div>
```

Comment:
This part I'm confused, why not use the `<Modal>` component here?

Reply:
This wasn't touched by this PR, just moved around.